--- config.def.h.orig	2022-05-19 20:48:58.180633260 +0300
+++ config.h	2022-05-18 21:33:13.018539330 +0300
@@ -1,4 +1,5 @@
 /* See LICENSE file for copyright and license details. */
+#include <X11/XF86keysym.h>
 
 /* appearance */
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
@@ -45,7 +46,7 @@
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -58,13 +59,24 @@
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *termcmd[]  = { "gnome-terminal", NULL };
+static const char *brightness_lower_cmd[] = { "brightness-", NULL };
+static const char *brightness_higher_cmd[] = { "brightness+", NULL };
+static const char *inc_volume[] = { "amixer", "set", "Master", "10%+" };
+static const char *dec_volume[] = { "amixer", "set", "Master", "10%-" };
+static const char *mute_volume[] = { "amixer", "set", "Master", "amixer -D pulse set Master 1+ toggle" };
+static const char *firefoxcmd[] = { "firefox", "NULL" };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+    { MODKEY|ShiftMask,             XK_b,      spawn,          {.v = firefoxcmd } },
+    { 0,                            XK_F2,     spawn,          {.v = brightness_lower_cmd } },
+    { 0,                            XK_F3,     spawn,          {.v = brightness_higher_cmd } },
+    { 0,                            XF86XK_AudioRaiseVolume, spawn, {.v = inc_volume } },
+    { 0,                            XF86XK_AudioLowerVolume, spawn, {.v = dec_volume } },
+    { 0,                            XF86XK_AudioMute,        spawn, {.v = mute_volume } },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
 	{ MODKEY,                       XK_i,      incnmaster,     {.i = +1 } },
